src/Turn.js

```diff
@@ -1,6 +1,30 @@
 class Turn {
-    constructor(guess) {
-        this.guess = guess
+    constructor(guess, playedCard) {
+        this.guess = guess;
+        this.playedCard = playedCard;
+    }
+    
+    returnGuess() {
+        return this.guess;
+    }
+
+    returnCard() {
+        return this.playedCard;
+    }
+
+    evaluateGuess() {
+        if (this.guess === this.playedCard.correctAnswer) {
+            return true;
+        }
+    }
+
+    giveFeedback() {
```

Comment:
Nice work with using the evaluateGuess function, instead of re-writing the logic.